src/layouts/index.css

```diff
@@ -1,52 +1,57 @@
 :root {
-  font-size: 16px;
+  font-size: 62.5%;
```

Comment:
if you want to use `rem` units, adding this line means that, by default, `rem` is just `px / 10`. So `3rem === 30px`. Really handy for translating from designs